markdown/ch6.markdown

```diff
@@ -359,41 +371,104 @@ x = Dinv2 @ eigvecs[:, 1]
 ```
 
 (The reasons for using this vector are too long to explain here, but appear in
-the paper's supplementary material, linked above.)
+the paper's supplementary material, linked above. The short version is that
+choosing this vector minimizes the total length of the links between neurons.)
+
+There is one small kink that we must address before proceeding: eigenvectors
+are only defined up to a multiplicative constant. This follows simply from the
```

Comment:
Define multiplicative constant (it because obvious shortly, but it's mean to throw in new terms without explaining them)